src/App.css

```diff
@@ -50,7 +50,7 @@ body {
 }
 
 .appbar.open {
-  background-color: rgba(255, 255, 255, .85);
+  opacity: .85;
```

Comment:
I don't understand why you are doing this. Why don't you just leave the default here and use an rgba color in the new overlayBackground to override it with any level of transparency you want? Using opacity will cause the icons in the appbar and the bottom bar to be transparent as well.